### PR TITLE
Makes chat auto fade away (and fixes CI)

### LINKED ISSUE #36: make chat auto fade away
just make the css element slowly decrease in height from the top downward until it disappears

### PR DESCRIPTION
**Pull request by Pandapip1**
I will test this later. Fixes #36

### CODE REVIEW

**Comment by Pandapip1** [comment]
To do:
- [x] Automatically hide chat messages after 8 seconds (5 seconds fully opaque, 3 seconds fading out)
- [ ] Show chat messages again when the chat window is open
- [ ] Code Formatting

**Comment by Pandapip1** [comment]
Testing URL: https://gracious-shannon-bd3372.netlify.app/

**Comment by AwesomestCode** [comment]
Ready for review?

**Comment by Pandapip1** [comment]
Updated testing link (apparently it only likes github pages): https://chatfade-pjs-test.pandapip1.com/

**Comment by Pandapip1** [comment]
For some reason, the CI seems to have broken. Does anyone have an idea on why this happened?

**Comment by AwesomestCode** [comment]
For reference, this is the log (I forgot if you can see it or not). I can't tell if something we changed recently broke it.
```ERROR in ./node_modules/jose/dist/browser/runtime/env.js 6:15-22
28
Module not found: Error: Can't resolve 'process/browser' in '/home/runner/work/prismarine-web-client/prismarine-web-client/node_modules/jose/dist/browser/runtime'
29
Did you mean 'browser.js'?
30
BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
31
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
32
The extension in the request is mandatory for it to be fully specified.
33
Add the extension to the request.
34
resolve 'process/browser' in '/home/runner/work/prismarine-web-client/prismarine-web-client/node_modules/jose/dist/browser/runtime'
35
  Parsed request is a module
36
  using description file: /home/runner/work/prismarine-web-client/prismarine-web-client/node_modules/jose/dist/browser/package.json (relative path: ./runtime)
37
    Field 'browser' doesn't contain a valid alias configuration
38
    resolve as module```

**Comment by Pandapip1** [comment]
Yea, it's definitely an issue with the CI. Here's a run that should have built: https://github.com/Pandapip1/prismarine-web-client/runs/4105482876

**Comment by Pandapip1** [comment]
Okay, I think it works. I'm marking it as ready for review.

**Comment by Pandapip1** [comment]
Here is the test URL again, for reference: https://chatfade-pjs-test.pandapip1.com/

Here is the issue that provided me with the fix to the CI: https://github.com/graphql/graphql-js/issues/2721

**Comment by Pandapip1** [comment]
It seems to work quite nicely!

File: lib/chat.js [hunk 1]
```diff
@@ -67,25 +66,44 @@ class ChatBox extends LitElement {
...
+        
+        .chat-message {
             display: block;
-        }    
+            background-color: rgba(0, 0, 0, 0.5);
+        }
+
+        .chat-message-fadeout {
+            opacity: 1;
+            transition: all 3s;
```
Comment (AwesomestAlt) [inline lib/chat.js:84]: I think this is a bit slow; other than that it seems good.

**Review by AwesomestAlt — APPROVED** [review]

**Review by AwesomestCode — APPROVED** [review]
Thank you for the contribution! The change I suggested earlier isn't really critical and is sort of a nitpick, so I'm merging now

**Comment by ngocloi93** [comment]
Okqy

**Pull request merged**